client/src/components/rules/Rules.tsx

```diff
@@ -1,29 +1,31 @@
 export const Rules = () => (
   <section className="section">
     <div>
-      <h1> Welcome to Pokémon Top Trumps! Gotta Catch ‘Em All!</h1>
+      <h1>Welcome to Pokémon Top Trumps! Gotta Catch ‘Em All!</h1>
     </div>
     <div>
-      <h2> Here are the rules to play. </h2>
-    </div>
-    <div> The game is out of 5 rounds.</div>
-    <div>Each player is given 5 random Pokémon cards. </div>
-    <div>
-      Each card has details about the Pokemon include 5 key stats that can be
-      compared. These stats are. Attack,Speed,Defence,Hit Point(hp),Weight
-    </div>
-    <div>
-      A player gets to turn their card first and pick an attribute to compare
-      against the other players.
+      <h2>Here are the rules to play.</h2>
     </div>
+    <ol>
```

Comment:
I thought this was in Mani's branch we've just merged? What's changed?

Reply:
It was, presumably it's because I merged main into the branch to resolve the conflicts?